patches/minecraft/net/minecraft/entity/monster/EntityMob.java.patch

```diff
@@ -0,0 +1,11 @@
+--- ../src-base/minecraft/net/minecraft/entity/monster/EntityMob.java
++++ ../src-work/minecraft/net/minecraft/entity/monster/EntityMob.java
+@@ -120,7 +120,7 @@
+                 ItemStack itemstack = this.func_184614_ca();
+                 ItemStack itemstack1 = entityplayer.func_184587_cr() ? entityplayer.func_184607_cu() : ItemStack.field_190927_a;
+ 
+-                if (!itemstack.func_190926_b() && !itemstack1.func_190926_b() && itemstack.func_77973_b() instanceof ItemAxe && itemstack1.func_77973_b() == Items.field_185159_cQ)
++                if (!itemstack.func_190926_b() && !itemstack1.func_190926_b() && itemstack.func_77973_b().canDisableShield(itemstack1, itemstack, p_70652_1_, this) && itemstack1.func_77973_b() == Items.field_185159_cQ)
```

Comment:
This is probably one of the rare cases where the two PRs should be merged as a general 'expand shield functionality' pr.